app/views/about/index.html.erb

```diff
@@ -32,8 +32,8 @@
                 <tr>
                   <td><%= link_to_if_authorized proxy.name, hash_for_edit_smart_proxy_path(proxy) %></td>
                   <td><%=h proxy.features.to_sentence %></td>
-                  <td><div class="proxy-status" data-url=<%= ping_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
-                  <td><div class="proxy-version" data-url=<%= version_smart_proxy_path(proxy) %>><%= _("Connecting..") %></div></td>
+                  <td><div class="proxy-status" data-url=<%= ping_smart_proxy_path(proxy) %>><%= image_tag '/assets/spinner.gif' %></div></td>
```

Comment:
1. why do you hardcode /assets? is that a new requirement?
2. why not use the css class with the spinner instead?